test_e2e.sh

```diff
@@ -691,6 +693,48 @@ fi
 # Wrap up
 pass
 
+##############################################
+# Test fetch skipping commit
+##############################################
+testcase "fetch-skip-depth-1"
+echo "$TESTCASE" > "$REPO"/file
+git -C "$REPO" commit -qam "$TESTCASE"
+GIT_SYNC \
+    --git="$SLOW_GIT_FETCH" \
+    --wait=0.1 \
+    --depth=1 \
+    --repo="file://$REPO" \
+    --branch=e2e-branch \
+    --rev=HEAD \
+    --root="$ROOT" \
+    --dest="link" \
+    > "$DIR"/log."$TESTCASE" 2>&1 &
+
+# wait for first sync which does a clone
+sleep 8
```

Comment:
comment that this spans a fetch, which is artificially slowed ?

Reply:
done.

Reply:
done.